### ISSUE TITLE
Please explain in README more about flashing stock Android

### ISSUE THREAD

**Issue by afcady** [DerTeufel/lineage_device_lenovo_x606fa#1]
I saw in the forums you asked a user if the version before installing was Android 10 based.

Is the requirement that Android 10(+) must be installed before installing this?

Can you explain the reason for the requirement?

Thank you.

**Comment by afcady** [comment]
Is it sufficient to select "convert to file encryption" in developer settings?  Or is that issue unrelated?

**Comment by DerTeufel** [comment]
it's already written in the readme, that you need to be on android10. as my device was already using android10 when i got it, i never could try anything android9 related (as downgrading seems to be causing some bricked devices).
Maybe it is also working when you try using it without installing android10 before, you're welcome to report your findings here, but i can't help you with this in any way.

**Comment by afcady** [comment]
My device came preinstalled with Android 10.  I initially tried to find the ROM for the exact version of Android 10 mentioned in the readme, but I couldn't find it (without paying money), and I was just as worried about trying with a newer version, than an older one.

Anyway, I went ahead and skipped upgrading.  But I did first convert to FBE.  Everything worked!

I will report that the version of Android that I had preinstalled on the tablet was: `TB-X606F_S300177_210304_BMP`

Also, `fastboot --wipe-and-use-fbe` failed (reporting `mke2fs failed with status 1`), but choosing `Convert to file encryption` in `Developer options` did succeed.  `fastboot -w` also failed with the same error but LineageOS still booted into the new user setup successfully.

Thanks for your reply and thank you even more for all your efforts making this work for everyone, friend!!
